### DISCUSSION TITLE
Add https://github.com/openlayers/ol-mapbox-style to TDesjardins/gwt-ol

### DISCUSSION THREAD

**Discussion by yhilem** [TDesjardins/gwt-ol3#175] [General]
I want use https://github.com/openlayers/ol-mapbox-style. I don't see how to integrate it into TDesjardins/gwt-ol. Thanks in advance.

**Comment by TDesjardins** [comment]
Hi @yhilem,

I think you need a thin wrapper to use the functionality of the ol-mapbox-style lib. For example you could wrap ol-mapbox-style/dist/stylefunction and use the gwt-ol classes for ol params in the method signature. To test and use you need to add the JS script of this lib to your HTML file.
From my point of view this could be an add-on to gwt-ol in a seperate Github repo.

**Reply by yhilem** [reply]
Thank you. Yes it would be good if it is an add-on to gwt-ol in a seperate Github repo.
I will try to do this (I don't know Javascript well) because I really need it.
